src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -55,24 +55,24 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by the user, or empty list if user has never posted a
    *     message. List is sorted by time descending.
    */
-  public List<Message> getMessages(String user) {
+  public List<Message> getMessages(String recipient) {
 
     List<Message> messages = new ArrayList<>();
 
     Query query =
         new Query("Message")
-            .setFilter(new Query.FilterPredicate("user", FilterOperator.EQUAL, user))
+            .setFilter(new Query.FilterPredicate("recipient", FilterOperator.EQUAL, recipient))
```

Comment:
Are you sure about this change?

Reply:
Yes, because we want to filter out the messages that are posted for the account of the page that we are looking at, not the messages that are only by/for the user.